emojione/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for emojione
+// Project: https://github.com/Ranks/emojione
+// Definitions by: Danilo Bargen <https://github.com/dbrgn/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var emojione: {
```

Comment:
Ah, that's nice.

I did not want to tie the definitions to the npm module because it's multiple megabytes and we just need a small subset of it. But with your way of exporting it should work for both cases, right?